src/main/frontend/pipeline-graph-view/pipeline-graph/main/PipelineGraphLayout.ts

```diff
@@ -238,10 +238,6 @@ function forEachChildStage(
     return;
   }
   for (const stage of topStage.children) {
-    // This can be in the graph if there is an unhandled exception.
-    if (stage.type === "PIPELINE_START") {
```

Comment:
and we definitely want it in the graph too, so you can click through from the job / build pages